x-pack/plugins/security_solution/public/entity_analytics/translations.ts

```diff
@@ -248,7 +248,7 @@ export const UPDATE_PANEL_GO_TO_DISMISS = i18n.translate(
 export const getMaxSpaceTitle = (maxSpaces: number) =>
   i18n.translate('xpack.securitySolution.riskScore.maxSpacePanel.title', {
     defaultMessage:
-      'Entity Risk Scoring in the current version can run in {maxSpaces} Kibana spaces.',
+      'Entity Risk Scoring in the current version can run in {maxSpaces} Kibana space.',
```

Comment:
Maybe we could consider "plural" form here to avoid changing translation in future when we update spaces count? Something like: `'Entity Risk Scoring in the current version can run in {maxSpaces} Kibana {maxSpaces, plural, =1 {space} other {spaces}}.'`.